delta-lake/common/src/main/databricks/scala/com/nvidia/spark/rapids/delta/DatabricksDeltaProviderBase.scala

```diff
@@ -107,9 +107,8 @@ trait DatabricksDeltaProviderBase extends DeltaProviderImplBase {
     }
   }
 
-  override def getReadFileFormat(format: FileFormat): FileFormat = {
-    val cpuFormat = format.asInstanceOf[DeltaParquetFileFormat]
-    GpuDeltaParquetFileFormat.convertToGpu(cpuFormat)
+  override def getReadFileFormat(relation: HadoopFsRelation): FileFormat = {
```

Comment:
Same question here. Does this work for all versions of deltalake on databricks? The file appears to be in a common directory.